arch/mips/soc/danube/Makefile

```diff
@@ -0,0 +1,17 @@
+soc-cflags := -I/$(srctree)/arch/$(ARCH)/soc/$(SOC_PATH)/ -I$(GCCMIPSMTI_TOOLCHAIN_PATH)/mips-mti-elf/include -EL -mips32r2
```

Comment:
Could you set some variable in the Makefile.toolchain* file so that you don't have to hardcode mips-mti-elf ?

Then this SOC wouldn't depend on a mips toolchain with this exact signature.

Makefile.toolchain.zephyr sets SYSROOT_TARGET_${ARCH} for this purpose.

Also, shouldn't the Makefile.toolchain.* file be putting this directory on the include path, and not the SoC?